Library/Homebrew/cmd/update.sh

```diff
@@ -754,16 +754,26 @@ EOS
   if [[ -n "${HOMEBREW_INSTALL_FROM_API}" ]]
   then
     mkdir -p "${HOMEBREW_CACHE}/api"
+    if [[ -f "${HOMEBREW_CACHE}/api/formula.json" ]]
+    then
+      INITIAL_JSON_SHASUM="$(shasum -a 256 "${HOMEBREW_CACHE}"/api/formula.json)"
```

Comment:
Thanks for the review; I will put the ruby script to check shasum256 instead of `shasum`.

Reply:
No, I think you want to do checksums instead of e.g. file sizes I think it's worth splitting out a `shasum` helper or something that we can use both here and in `vendor-install`.

Reply:
I will compare the file size by  `ls -s "${HOMEBREW_CACHE}"/api/formula.json)"`.

Reply:
@hyuraku actually the best bet would be `wc -c "${HOMEBREW_CACHE}"/api/formula.json)"`